app/web/components/HorizontalScroller.tsx

```diff
@@ -13,7 +13,7 @@ const useStyles = makeStyles((theme) => ({
     display: "inline-flex",
     flexDirection: "row",
     height: "100%",
-    width: "100%",
+    width: "100vw",
```

Comment:
Why was this changed? This could cause horizontal overflow issues if used in a container with a margin, though I do believe we always use it with 0 margin atm

Reply:
Yeah, that's something we need to be careful with, and maybe I need to test more across browsers and everywhere this is used.

With `100%`, when using `HorizontalScroller` in the dashboard or any other view that has the app layout (because it has paddings on the side), this is what it looks like:


https://user-images.githubusercontent.com/1557348/166420702-306921ea-fe1c-4ac9-97ba-98193b20f1cc.mov


The tricky thing of this use case is that the horizontal scroller class name is also combined with class names passed. Here, `classes.upcomingEventContainer` includes a `100vw` rule that is being overwritten. 

<img width="385" alt="Screen Shot 2022-05-03 at 09 54 39" src="https://user-images.githubusercontent.com/1557348/166420716-b55ce822-29ef-4a7b-9b50-f5dd45417a79.png">

Also, `classes.cardContainer` seems to be setting the scroller to be centered and bypassing the paddings. It seems that these 3 sets of styles (`classes.cardContainer`, `classes.upcomingEventContainer`, and horizontal scroller's `classes.root`) are conflicting a bit with each other.

<img width="402" alt="Screen Shot 2022-05-03 at 09 58 01" src="https://user-images.githubusercontent.com/1557348/166420849-9230cf63-0194-484c-9fe4-4060af3c57f8.png">

Maybe for now I can leave it with `100%`, despite the visual bug, and we tackle later refactoring the horizontal scroller so it's more self-contained and it works without needing to pass any styles from outside it? How does that sound?

Reply:
Thanks for the explanation! Just had a look.

I think your change is indeed the best/easiest solution, this component is always supposed to be full-width on mobile.

Also seems to me like we can get rid of `classes.upcomingEventContainer` completely since the only line is overriden or has no effect at any breakpoint anyway as far as I can tell.